src/App.jsx

```diff
@@ -13,41 +13,15 @@ import { useStateWithStorage } from './utils';
 
 export function App() {
 	const [data, setData] = useState([]);
-	/**
-	 * Here, we're using a custom hook to create `listToken` and a function
-	 * that can be used to update `listToken` later.
-	 *
-	 * `listToken` is `my test list` by default so you can see the list
-	 * of items that was prepopulated for this project.
-	 * You'll later set it to `null` by default (since new users do not
-	 * have tokens), and use `setListToken` when you allow a user
-	 * to create and join a new list.
-	 */
```

Comment:
Thank you for thinking ahead!  Removing comments as we go is such a good practice.